GVFS/GVFS.Common/Git/GitAuthentication.cs

```diff
@@ -152,6 +164,28 @@ public bool TryInitializeAndRequireAuth(ITracer tracer, out string errorMessage)
             return false;
         }
 
+        private bool TryParseCachedAuthString(out string username, out string password)
+        {
+            if (this.cachedAuthString != null)
+            {
+                byte[] data = Convert.FromBase64String(this.cachedAuthString);
+                string credentialString = Encoding.ASCII.GetString(data);
+
+                string[] foo = credentialString.Split(':');

```

Comment:
nit: I think we use more descriptive names than foo generally :-)

Reply:
Yeah.. I missed this one! Now you know how unimaginative my variable names are when doing WIP.
(Sometimes I also use `x`, `soup` or `fish`!)

Reply:
Hahahaha I love soup! :-)